vuepress/docs/next/tutorials/create/mfe/communication.md

```diff
@@ -265,392 +223,51 @@ const widgetEvent = new CustomEvent('greeting', {
-- `publisher-widget/static/js/main.js`
-- `publisher-widget/static/js/runtime.js`
-- `publisher-widget/static/js/vendor.js`
-
-#### Subscriber Widget
-
-- Repeat steps 1-8 for the `subscriber-widget`.
-
-### Add Widgets
-
-#### Publisher Widget
-
-1. Go to `Entando App Builder` in your browser.
-
-2. Go to `Components` → `Micro Frontends & Widgets` at the top nav.
-
-3. Click `Add`.
-
-4. Enter the following:
-
-- `Code: publisher_widget` → note: dashes are not allowed
-- `Title: Publisher Widget` → for both English and Italian languages
-- `Group: Free Access`
-- `Custom UI:`
-
-``` ftl
-<#assign wp=JspTaglibs[ "/aps-core"]>
-<link rel="stylesheet" type="text/css" href="<@wp.resourceURL />publisher-widget/static/css/main.css">
-<script async src="<@wp.resourceURL />publisher-widget/static/js/runtime.js"></script>
-<script async src="<@wp.resourceURL />publisher-widget/static/js/vendor.js"></script>
-<script async src="<@wp.resourceURL />publisher-widget/static/js/main.js"></script>
-<publisher-widget />
-```
-
-5. Click `Save`.
-
-#### Subscriber Widget
-
-Repeat steps 1-5 for the subscriber widget.
-
-- `Code: subscriber_widget` → note: dashes are not allowed
-- `Title: Subscriber Widget` → for both English and Italian languages
-- `Group: Free Access`
-- `Custom UI:`
-
-``` ftl
-<#assign wp=JspTaglibs[ "/aps-core"]>
-<link rel="stylesheet" type="text/css" href="<@wp.resourceURL />subscriber-widget/static/css/main.css">
-<script async src="<@wp.resourceURL />subscriber-widget/static/js/runtime.js"></script>
-<script async src="<@wp.resourceURL />subscriber-widget/static/js/vendor.js"></script>
-<script async src="<@wp.resourceURL />subscriber-widget/static/js/main.js"></script>
-<subscriber-widget />
-```
+Refer to the tutorial on how to [publish a bundle project](../pb/publish-project-bundle.md#create-and-deploy-a-bundle-project) for more detailed instructions.
 
 ### View on a Page
-You can setup the widgets on an existing page (such as the Home page) or [create your own page](../../compose/page-management.md). The following steps assume you'll use the Home page.
 
-1. Go to `Pages` → `Management`
+Set up the widgets on an existing page or [create your own page](../../compose/page-management.md). The following steps assume you'll use the `Home` page.
 
-2. For the `Home` page `(folder icon)`, in the `Actions` column, click the `⋮` icon
+1. Go to `Pages` → `Management`
 
-3. Click `Edit`.
+2. On the line item for the `Home` page, in the `Actions` column, click the `⋮` icon
 
-4. In the `Settings` section, select a Page Template with more than one frame, e.g. `1-column`:
+3. Click `Edit`
 
-- `Page Template: 1 Column`
+4. In the `Settings` section, select a Page Template with more than one frame
 
-5. Click `Save and Configure`.
+5. Click `Save and Design`
 
-6. In the `WIDGETS` sidebar on the right:
+6. From the `Widgets` tab in the right sidebar, drag your publisher and subscriber widgets into `Frame 1` and `Frame 2`
```

Comment:
drag >> drag and drop
it doesn't matter which goes in which frame, right?

Reply:
dragging the widget into the frame performs a drag and drop

it doesn't. the way we read from left to right, top to bottom would map to publisher on left/top but that's not worth mentioning

Reply:
not everyone reads left to right. I would totally read this one to mean put it in any frame

Reply:
true, all the more reason to omit specific placement from the instruction. placement has no impact on functionality and is not subject to any constraints; the user is free to place the widgets in the layout of their choice. because the widgets exhibit some causality, the user will likely place them according to convention for cause and effect